Makefile

```diff
@@ -166,3 +166,7 @@ build-docker-release-main:
 push-docker-release-main:
 	$(info ETCD_VERSION: $(ETCD_VERSION))
 	docker push gcr.io/etcd-development/etcd:$(ETCD_VERSION)
+
+lint:
```

Comment:
Could you add a rule to fix issues? It would just run `golangci-lint run --fix`

Reply:
I mention in the commit message that "auto fixing" should be left for a later commit. Once we have decided that we have the right `.golangci.yml` for us.